tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/controllers/EndringsmeldingArrangorController.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.amt.tiltak.deltaker.controllers
+
+import no.nav.amt.tiltak.common.auth.AuthService
+import no.nav.amt.tiltak.common.auth.Issuer
+import no.nav.amt.tiltak.core.port.ArrangorAnsattTilgangService
+import no.nav.amt.tiltak.core.port.DeltakerService
+import no.nav.amt.tiltak.deltaker.service.EndringsmeldingService
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.*
+import java.time.LocalDate
+import java.util.*
+
+@RestController
+@RequestMapping(value = ["/api/tiltakarrangor/endringsmelding"])
+class EndringsmeldingArrangorController(
+	private val endringsmeldingService: EndringsmeldingService,
+	private val arrangorTilgangService: ArrangorAnsattTilgangService,
+	private val deltakerService: DeltakerService,
+	private val authService: AuthService
+	) {
+
+	@ProtectedWithClaims(issuer = Issuer.TOKEN_X)
+	@PostMapping("/deltaker/{deltakerId}/startdato")
+	fun registrereStartDato(@PathVariable("deltakerId") deltakerId: UUID, @RequestParam startDato: LocalDate) {
```

Comment:
Flisespikkeri: `registrereStartDato` -> `registrerStartDato`

Reply:
Usikker på om det er en konvensjon å bruke body istedenfor query params når man poster data, selv med kun 1 parameter

Reply:
Det er mer logisk for meg at man gjør sånn av en eller annen grunn, at verdiene man poster er parametre